python/lsst/ts/m2gui/utils.py

```diff
@@ -222,3 +224,58 @@ def get_num_actuator_ring(ring):
         return 18
     else:
         raise ValueError(f"Not supported ring: {ring!r}")
+
+
+def prompt_dialog_warning(description, is_prompted=True):
+    """Prompt the warning dialog.
+
+    The user must react to this dialog. The rest of the GUI is blocked until
+    the dialog is dismissed.
+
+    Parameters
+    ----------
+    description : `str`
+        Description that will be shown in the dialog.
+    is_prompted : `bool`, optional
+        Dialog will be prompted or not. The value of False is used in the unit
```

Comment:
Better "When False, dialog will not be executed. That is used for tests. Default is True, which shall be the case when used in real GUI."?

Reply:
Done in the commit: [b2dae7c](https://github.com/lsst-ts/ts_m2gui/pull/10/commits/b2dae7c31b0f66668b2974b6761c5d7795e35f4b). Thanks!